eternalcore-api/src/main/java/com/eternalcode/core/feature/ignore/event/PurgeEvent.java

```diff
@@ -0,0 +1,57 @@
+package com.eternalcode.core.feature.ignore.event;
+
+import org.bukkit.command.CommandSender;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+
+public class PurgeEvent extends Event implements Cancellable {
```

Comment:
I don't think it is good to create event for purge. But please rename it to mark it as event from ingore domain.

Reply:
Noted, I'll consider removing it completely as it is a low-level action.